pkg/storage/cassandra_dependencies.go

```diff
@@ -36,7 +37,7 @@ func cassandraDeps(jaeger *v1.Jaeger) []batchv1.Job {
 	}
 
 	if jaeger.Spec.Storage.CassandraCreateSchema.Image == "" {
-		jaeger.Spec.Storage.CassandraCreateSchema.Image = fmt.Sprintf("%s:%s", viper.GetString("jaeger-cassandra-schema-image"), viper.GetString("jaeger-version"))
+		jaeger.Spec.Storage.CassandraCreateSchema.Image = fmt.Sprintf("%s:%s", viper.GetString("jaeger-cassandra-schema-image"), version.Get().Jaeger)
```

Comment:
Shouldn't this use the same approach as has been changed in the above files - don't modify the CR spec, and instead use local var initialised using the viper property?

Reply:
Yes, good catch!